### PR TITLE
Revert reverted e2e changes for MicroShift

### PR DESCRIPTION
**Pull request by ingvagabund**
Reverted by https://github.com/openshift/origin/pull/27499

### CODE REVIEW

**Comment by ingvagabund** [comment]
`error: unable to create a release: operator "cluster-capi-operator" contained an invalid image-references file: no input image tag named "ibmcloud-cluster-api-controllers"`

File: test/extended/networking/services.go [hunk 1]
```diff
@@ -165,7 +165,7 @@ var _ = Describe("[sig-network] services", func() {
 	})
 
 	InBareMetalIPv4ClusterContext(oc, func() {
-		It("ensures external auto assign cidr is configured correctly on the cluster [Serial]", func() {
+		It("ensures external auto assign cidr is configured correctly on the cluster [apigroup:config.openshift.io] [Serial]", func() {
```
Comment (atiratree) [inline test/extended/networking/services.go:168]: this group is not present in microshift, right? FYI this test should be skipped there nevertheless

Comment (atiratree) [reply test/extended/networking/services.go:168]: just checked - it is not present

**Comment by ingvagabund** [comment]
Both `[sig-devex][Feature:ImageEcosystem][mysql][Slow] openshift mysql image Creating from a template should instantiate the template [apigroup:apps.openshift.io]` and `[sig-devex][Feature:ImageEcosystem][mariadb][Slow] openshift mariadb image Creating from a template should instantiate the template [apigroup:image.openshift.io][apigroup:operator.openshift.io][apigroup:config.openshift.io][apigroup:apps.openshift.io]` are repeatedly failing in other PRs as well:
- https://github.com/openshift/origin/pull/27525
- https://github.com/openshift/origin/pull/27523
- https://github.com/openshift/origin/pull/27521

Based on https://prow.ci.openshift.org/pr-history/?org=openshift&repo=origin&pr=27502 it looks like `e2e-gcp-ovn-image-ecosystem` started running just recently.

**Comment by ingvagabund** [comment]
Waiting for https://github.com/openshift/origin/pull/27529 to merge

**Comment by ingvagabund** [comment]
Merged through https://github.com/openshift/origin/pull/27540.

**Pull request closed without merging**